datafusion/physical-expr/src/scalar_function.rs

```diff
@@ -37,7 +37,7 @@ use std::sync::Arc;
 use arrow::datatypes::{DataType, Schema};
 use arrow::record_batch::RecordBatch;
 
-use datafusion_common::Result;
+use datafusion_common::{internal_err, Result};
```

Comment:
This is the change from https://github.com/apache/datafusion/pull/10317 to get the CI to pass